CookieTTS/_1_preprocess/scripts/audio_preprocessing.py

```diff
@@ -70,9 +70,9 @@ def normalize_volumes_mixmode(directory, amplitude=0.08, ext='.wav'):
     RETURNS:
         None
     """
-    subdirectories = [x[0] for x in os.walk(directory)]
-    for subdirectory in subdirectories:
-        os.system(f"normalize-audio -w 16 -a {amplitude} -b '{subdirectory}/'*{ext}")
```

Comment:
Can you confirm the reason for this change?
http://normalize.nongnu.org/README.html
This was intended to be used with "mixmode" which requires all the files in a directory to be called together.

Reply:
Actually I believe I've misread the output. It only normalised one file and the non-recursiveness of normalize-audio confused me a bit, but I added verbosity and saw that it checked every file and you check recursively manually. I'll cherrypick the other change into a new PR.